Library/Homebrew/utils/io_selector.rb

```diff
@@ -0,0 +1,79 @@
+require "delegate"
+require "English"
```

Comment:
This shouldn't be required; it's part of `global` now. It may not be needed in the `spec_helper` either.